### PR TITLE
ci: add multiple clusters support

### LINKED ISSUE #745: e2e: add support for multiple clusters
In the current E2E tests only one Elemental cluster is deployed, so we need to add the support for multiple cluster.

### PR DESCRIPTION
**Pull request by ldevulder**
Will fix https://github.com/rancher/elemental/issues/745.

Verification runs:
- [CLI-K3s-Multi_Cluster-RM_Stable](https://github.com/rancher/elemental/actions/runs/6694160766/job/18186841681?pr=1007)

Regression runs:
- [UI-K3s-RM_Stable](https://github.com/rancher/elemental/actions/runs/6695175400)
- [UI-RKE2-RM_Stable](https://github.com/rancher/elemental/actions/runs/6693672442)
- [CLI-K3s-RM_Stable](https://github.com/rancher/elemental/actions/runs/6693655110)
- [CLI-RKE2-RM_Stable](https://github.com/rancher/elemental/actions/runs/6695177519)

### CODE REVIEW

File: .github/workflows/cli-k3s-multi_cluster-rm_stable.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      # !! DEBUG !!
+      destroy_runner: false
+      iso_boot: true
+      k8s_version_to_provision: v1.26.8+k3s1
+      # !! DEBUG !!
+      operator_repo: oci://registry.suse.com/rancher
+      os_to_test: stable
+      # !! DEBUG !!
+      runner_template: elemental-e2e-ci-runner-spot-x86-64-template-n2-highmem-64-v4
+      test_type: multi
```
Comment (juadk) [inline .github/workflows/cli-k3s-multi_cluster-rm_stable.yaml]: As we use this variable to difference `ui` and `cli` tests, could you use `multi_cli` instead?

Comment (ldevulder) [reply .github/workflows/cli-k3s-multi_cluster-rm_stable.yaml]: Yes, I split to `single_cli` and `multi_cli` (in addition to the already existing value `ui`).

File: .github/workflows/cli-k3s-multi_cluster-rm_stable.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
+    with:
+      test_description: "CI/Manual - CLI - MultiCluster - Deployment test with Standard K3s"
+      cluster_name: cluster-k3s
+      cluster_number: 40
+      destroy_runner: ${{ github.event_name == 'schedule' && true || inputs.destroy_runner }}
+      iso_boot: true
+      k8s_version_to_provision: v1.26.8+k3s1
+      runner_template: elemental-e2e-ci-runner-spot-x86-64-template-n2-highmem-64-v4
+      test_type: multi
```
Comment (juadk) [inline .github/workflows/cli-k3s-multi_cluster-rm_stable.yaml]: Should be `multi_cli` I think

File: .github/workflows/cli-rke2-multi_cluster-rm_stable.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    with:
+      test_description: "CI/Manual - CLI - MultiCluster - Deployment test with Standard RKE2"
+      ca_type: private
+      cluster_name: cluster-rke2
+      cluster_number: 40
+      destroy_runner: ${{ github.event_name == 'schedule' && true || inputs.destroy_runner }}
+      iso_boot: true
+      k8s_version_to_provision: v1.26.8+rke2r1
+      runner_template: elemental-e2e-ci-runner-spot-x86-64-template-n2-highmem-64-v4
+      test_type: multi
```
Comment (juadk) [inline .github/workflows/cli-rke2-multi_cluster-rm_stable.yaml]: Should be `multi_cli` I think

**Review by juadk — CHANGES_REQUESTED** [review]

File: .github/workflows/cli-k3s-multi_cluster-rm_stable.yaml (hunk 2 above)
Comment (ldevulder) [reply .github/workflows/cli-k3s-multi_cluster-rm_stable.yaml]: Yes, fixed, thanks!

File: .github/workflows/cli-rke2-multi_cluster-rm_stable.yaml (hunk 3 above)
Comment (ldevulder) [reply .github/workflows/cli-rke2-multi_cluster-rm_stable.yaml]: Yes, fixed, thanks!

**Review by juadk — APPROVED** [review]
LGTM

**Pull request merged**